docs/architecture/psa-thread-safety.md

```diff
@@ -1,9 +1,16 @@
-Thread safety of the PSA subsystem
-==================================
+# Thread safety of the PSA subsystem
 
-## Requirements
+Currently PSA Crypto API calls in Mbed TLS releases are not thread-safe. In Mbed TLS 3.6 we are planning to add a minimal support for thread-safety of the PSA Crypto API (see #strategy-for-3.6).
```

Comment:
`#strategy-for-3.6` is not a hyperlink in either pandoc or github markdown. Please fix the intended hyperlinks to work in these two environments.